packages/react-tc-ui-kit/src/components/Textarea/Textarea.js

```diff
@@ -0,0 +1,93 @@
+import { useEffect, useRef } from 'react';
+
+import './Textarea.css';
+
+const DEFAULT_TEXTAREA_HEIGHT = 120;
+
+export default function Textarea({
+  inputRef,
+  style,
+  id,
+  className = '',
+  label,
+  disabled,
+  value = '',
+  maxLength,
```

Comment:
we need to handle case if this prop not passed
hide counter and check max-length not setting for textarea